rd_ui/app/scripts/directives/plotly.js

```diff
@@ -121,6 +121,8 @@
   angular.module('plotly', [])
     .constant('ColorPalette', ColorPalette)
     .directive('plotlyChart', function () {
+      var baseHeight = 250;
```

Comment:
Another option is to skip setting height here and delegate this to Plotly (by setting the height of the div and only specifying the margin size).